.github/workflows/release.yaml

```diff
@@ -23,4 +23,4 @@ jobs:
             ## CHANGELOG
             See [CHANGELOG](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/CHANGELOG-0.x.md) for full list of changes
           draft: false
-          prerelease: false
+          prerelease: true
```

Comment:
Curious about this change. It seems unusual to flag a semver stable tag as a prerelease.

Reply:
I think the quick fix of this would be to have the job set `draft: true` rather than `prerelease: true`.